### PR TITLE
Create ProgramRituals.md

### PR DESCRIPTION
**Pull request by suprenant**
Taking a first stab at getting our SAFe processes into Github. Pulling from this Google Doc: 
https://docs.google.com/document/d/1prGeLbcZ5_gNtLeeQyNNL8lmLvStGcVeYBu9cWqhgPk/edit

There's a lot of content here, so I created a new file for now, but we may want to fold this into DeliveryProcess.md or possibly pull in some content from there into this new doc.

Once these next two items are done (thinking in the next day), this draft will be ready for review: 
 - Planning Increment (PI)
 - Planning Increment (PI) planning

Past these, I'd like to do one more overall review and take a deeper look at the Inspect and Adapt section as time allows.

### CODE REVIEW

File: ProgramRituals.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+# Program Rituals and Practices
+
+## Scaled Agile Framework (SAFe)
+
+Managing a small team for a short project can be challenging and require practices, discipline, and experience to run effectively.
+
+When the team size and project duration expand, new sets of practices are required to both coordinate the efforts of a larger team (often broken down into a subset of teams) and regularly set, evaluate, and reset objectives over a period of time to insure cross-team collaboration and that the team regularly prioritizes the highest-value work while adapting to change, including shifts in business context, personnel churn, and other factors that may influence capacity and priorities.
+
+On cloud.gov, which has now run for several years with a team size varying from twelve to eighteen members, we use a set of [Scaled Agile Framework (SAFe)](http://www.scaledagileframework.com/guidance-essential-safe/) practices with other Agile techniques to help meet these needs:
```
Comment (mogul) [inline ProgramRituals.md]: s/set of/subset of the/

**Comment by suprenant** [comment]
OK, all sections have been filled in, and this is ready for review. @mogul, want to take another look?

Once this is done, we can possibly link to this from DeliveryProcess.md and/or check that document to see if some of the content that is currently there would be better placed in this file. 

Or, we can move all of this over there, but that would seem like a really long document to read.

**Comment by mogul** [comment]
Made a bunch of edits... Enough that someone else should probably review. Maybe @nikzei? I think this is mostly good to read as a blog post! Next steps would be to peel all the stuff about feature management out of the other page and put it in its own section here.

**Comment by suprenant** [comment]
Cool. @nikzei if you want to review this, I can then pick up the peeling of feature management steps that @mogul is suggesting here.

**Pull request merged**